hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/FindMissingPadding.java

```diff
@@ -96,7 +95,7 @@ public class FindMissingPadding extends Handler implements SubcommandWithParent
   private final Set<OzoneKey> affectedKeys = new HashSet<>();
 
   @Override
-  protected OzoneAddress getAddress() throws OzoneClientException {
+  protected OzoneAddress getAddress() throws IOException {
```

Comment:
I think the more specific exception should be kept in `throws` for all methods that do not otherwise throw `IOException`.

Reply:
Makes sense. I will update the patch.